js/viz/landscape_sst.js

```diff
@@ -89,7 +89,7 @@ export const landscape_sst = async (
 
     viz_state.cats.square_tile_size = square_tile_size
 
-    await set_meta_gene(viz_state.genes, base_url)
+    await set_meta_gene(viz_state.genes, base_url, viz_state.seg.version)
```

Comment:
lets not modify landscape_sst right now. Lets set a default value for set_meta_gene